### ISSUE TITLE
Allow heroku free plans for opensource projects that heroku itselfs benefits from

### ISSUE THREAD

**Issue by WolfgangFahl** [heroku/roadmap#126]
### Required Terms

- [X] I agree to follow this project's [Code of Conduct](https://github.com/heroku/roadmap/blob/main/CODE_OF_CONDUCT.md)
- [X] I have read and accept the [Salesforce Program Agreement](https://www.salesforce.com/company/program-agreement/)

### What service(s) is this request for?

Heroku

### Tell us about what you're trying to solve. What challenges are you facing?

```markdown
https://justpy-demo.herokuapp.com/ doesn't work any more and you end up getting an error message

at=error code=H14 desc="No web processes running"
```
which leads to https://stackoverflow.com/questions/41804507/h14-error-in-heroku-no-web-processes-running

And the solutions there don't work any more. The hint
```
▸    Subscribe to Eco to scale your dynos. Learn more at
 ▸    https://blog.heroku.com/new-low-cost-plans
```
Does not help here. Especially since https://www.heroku.com/pricing doesn't show you any pricing as of 2022-12-18
![grafik](https://user-images.githubusercontent.com/1336221/208305099-2af61ee6-d41c-4a97-b081-f42ffabc6b49.png)

justpy is an open source project with roughly 1000 stars and we do have a specific heroku tutorial
https://justpy.io/blog/heroku/

So basically we would be doing advertising for heroku if it would work. Why should we now paying for doing this advertising service instead of getting permanently supported by heroku for doing this advertising service?

Instead now the demo doesn't work and this looks as if heroku is broken and i assume has a negative impact on herokus business.
```

**Comment by friism** [comment]
Thanks, this is something we're looking into. It will likely take the form of credits applied to Heroku accounts used by open source projects

**Comment by WolfgangFahl** [comment]
see also https://github.com/justpy-org/justpy/discussions/649

**Comment by Jackenmen** [comment]
Free credits that have been granted during the previous round have just expired today so it would be good to get an update on this very soon. Emails to ospo-heroku-credits email currently result in the auto-responder message:
> Hello!
> 
> Thank you for reaching out to the Heroku OSS Credits. At this time our Heroku Credits Request form is closed. We will be reaching out directly to the projects that are receiving credits this round.
> 
> If you are interested in applying for Heroku Credits during the next round please follow @Heroku on Twitter.
> 
> Thank you!

According to the email from November, Heroku expected to have more information on the free credit program by February 2023 so the lack of any information is a bit concerning.

**Comment by WolfgangFahl** [comment]
Currently this is quite a showstoppe for the progress in our project since an important milestone is that we show that all demos from our tutorials are working.

**Comment by friism** [comment]
Hey, we just got this program off the ground details here: https://www.heroku.com/open-source-credit-program

**Comment by andre5oto** [comment]
We are announcing a Heroku credits program for open source projects starting in March 2023. See our [Heroku 2022 Roundup blog](https://blog.heroku.com/heroku-2022-roundup#supporting-open-source-projects) for more information.

**Comment by friism** [comment]
Details from the blog post https://www.heroku.com/open-source-credit-program

**Comment by WolfgangFahl** [comment]
Why is this closed justpy can't use heroku apps since the application process is not finished and we have therefore moved to http://justpy-demo.bitplan.com instead and the solution of https://stackoverflow.com/a/70232126/1497139 which is hosted on heroku doesn't work any more and so are probably lots of other little goodies that the open source world has created trusting that there would be a free low usage profile available on heroku.

**Comment by friism** [comment]
@WolfgangFahl I closed the issue because our open source credit program is now in place. I'm sorry the application process takes a while, but I'm happy you found different hosting in the interim.
